### DISCUSSION TITLE
SEND transaction become RECEIVE after resync wallet from scratch

### DISCUSSION THREAD

**Discussion by willy-tedyanto** [Chia-Network/chia-blockchain#9024] [General]
I have setup Chia testnet7 to develop an app. I use it to test send and receive test XCH (TXCH).

I found something strange yesterday when I try to add all INCOMING transaction and all OUTGOING transaction from console (chia wallet get_transactions). All INCOMING - all OUTGOING should be the WALLET BALANCE. But that's not the case, when I subtract ALL INCOMING with ALL OUTGOING. The value is MINUS (-76.88 TXCH), while wallet balance showed (1.018 TXCH).

I guessed maybe my wallet db was corrupt, so I moved original wallet to another folder and start syncing wallet again. This morning, it has completed syncing and when I run : chia wallet get_transactions , all my transactions become INCOMING. No OUTGOING transactions anymore.

I sum all INCOMING transactions, and it shows 88 .56 TXCH.

I have updated to Chia 1.2.10 prior to syncing wallet from scratch. Have any of you ever experienced something like this ? Or if any Chia dev could give me clue ? Thank you in advance.

**Comment by loppefaaret** [comment]
It is the way the wallet handles coin ID's - it would add significant exxtra processing to track outgoing transactions, instead of just the fact that they aren't belonging to the wallet anymore.
When you resync, it will not list any previously made outgoing transactions, only incoming. which is what you are seeing now - the spendable balance is correct
if this does not answer your question fully, i can suggest coming by our keybase chat, and pose it there, where it will be more visible for people able to explain it better than i am.
